app/screens/edit_profile/edit_profile.js

```diff
@@ -87,16 +88,29 @@ export default class EditProfile extends PureComponent {
         props.navigator.setOnNavigatorEvent(this.onNavigatorEvent);
         props.navigator.setButtons(buttons);
 
+        const profileImageUri = '';
+
         this.state = {
             email,
             firstName,
             lastName,
             nickname,
             position,
             username,
+            profileImageUri,
         };
     }
 
+    getProfileImageUri = async (id, lastPictureUpdate) => {
```

Comment:
The async/await and the try/catch aren't actually needed here since `Client4.getProfilePictureUrl` is synchronous